packages/rocketchat-livechat/imports/server/rest/messages.js

```diff
@@ -46,7 +46,8 @@ RocketChat.API.v1.addRoute('livechat/messages', { authRequired: true }, {
 					_id: Random.id(),
 					rid,
 					token: visitorToken,
-					msg: message.msg
+					msg: message.msg,
+					receivedByAPI: true
```

Comment:
What do you think on replacing `receivedByAPI: true` to `sender: 'API'`?

Reply:
@renatobecker , that's better.
But, `sender` can cause confusion with the user that sent the message.
What if we use `channel: 'API'`?

Reply:
@hmagarotto, Yes, I understand your point.. The problem using `channel` is that every `room` is a Rocket.Chat channel .. =/
Maybe `provider: 'API'`, what do you think?

Reply:
Good! I'll make the changes.